lib/wizrb/lighting/state.rb

```diff
@@ -40,11 +40,7 @@ def color_temp
       end
 
       def brightness
-        if @state[:dimming] <= 10
-          @state[:dimming].to_i * 10
-        else
-          @state[:dimming]
-        end
+        @state[:dimming]
```

Comment:
There's actually a problem with this change. The lights are programmed weird where you can set the brightness from min to max with two ranges:
1. Float values 1.0 - 10.0
2. Integer values 10 - 100

It's super weird, I know. Though since it is a feature of the bulbs, I decided to support it by mapping the float value to the integer value. This prevents bulbs that have a state of 1.0 - 10.0 from making the user of this library handle that weird edge case. Give it a shot, you'll see what I mean:
```ruby
# Using Floats
light.dispatch_event(Wizrb::Lighting::Events::Event.new(method: 'setState', params: { dimming: 0.0 })) # Errors
light.dispatch_event(Wizrb::Lighting::Events::Event.new(method: 'setState', params: { dimming: 1.0 })) # OK - Min
light.dispatch_event(Wizrb::Lighting::Events::Event.new(method: 'setState', params: { dimming: 10.0 })) # OK - Max
light.dispatch_event(Wizrb::Lighting::Events::Event.new(method: 'setState', params: { dimming: 11.0 })) # Errors

# Using Integers
light.dispatch_event(Wizrb::Lighting::Events::Event.new(method: 'setState', params: { dimming: 9 })) # Errors
light.dispatch_event(Wizrb::Lighting::Events::Event.new(method: 'setState', params: { dimming: 10 })) # OK - Min
light.dispatch_event(Wizrb::Lighting::Events::Event.new(method: 'setState', params: { dimming: 100 })) # OK - Max
light.dispatch_event(Wizrb::Lighting::Events::Event.new(method: 'setState', params: { dimming: 101 })) # Errors
```

As for the Smart Plug, I did put that in the "Unsupported Devices" section since I don't have one to test with. I am working on a big overhaul that adds support for the plug and moves basic classes into a shared resource that can be inherited by multiple device types.

Reply:
I tried this out and you're right. Very weird behavior.

However, when I set dimming to a float value, lets's say `2.5` and call refresh, then the dimming attribute in the getPilot response is `25`. I never get a float value or a value below 10. So not sure if the conversion is really needed?

The Smart Plug doesn't return the dimming attribute so a nil case should be handled in `State#brightness`. I own the Smart Plug Type F and powering on/off just works.

Looking forward to the overhaul, if I can help you with testing then let me know!

Reply:
Hey there, long time no see! I've been pretty busy with work, but here's the changes I promised:
https://github.com/bert-mccutchen/wizrb/pull/4